deliver/lib/deliver/runner.rb

```diff
@@ -86,10 +88,15 @@ def verify_version
 
     # Upload all metadata, screenshots, pricing information, etc. to iTunes Connect
     def upload_metadata
+      upload_metadata = UploadMetadata.new
+      upload_screenshots = UploadScreenshots.new
+
       # First, collect all the things for the HTML Report
-      screenshots = UploadScreenshots.new.collect_screenshots(options)
-      UploadMetadata.new.load_from_filesystem(options)
-      UploadMetadata.new.assign_defaults(options)
+      screenshots = upload_screenshots.collect_screenshots(options)
+      upload_metadata.load_from_filesystem(options)
```

Comment:
I'm wondering if we can pass the options here into the `UploadMetadata` and `UploadScreenshot` initializers?

Feel free to push back on that, just might be nice to save a few lines here, also because below on lines 99, 108 and 109, those objects are using the options again?

This shouldn't be a blocker, just something to think about 😄